cpp/src/aztec3/circuits/abis/barretenberg/aggregation_object.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
```

Comment:
I've created a fresh PR for this: https://github.com/AztecProtocol/barretenberg/pull/257 because this should not live in a3-circuits.